### PR TITLE
[OTA-3291] Improved campaign stats

### PR DESCRIPTION
**Pull request by piotrsmol**

### CODE REVIEW

File: ota-plus-web/app/reactapp/src/components/campaigns/CampaignSummary.jsx [hunk 1]
```diff
@@ -66,43 +80,51 @@ class CampaignSummary extends Component {
         {notInPreparation && (
           <div className="campaigns__column" id={`campaign-failed-${campaign.id}`}>
             <span>
-              <span>{`${failedRate} %`}</span>
+              <Tooltip title={t('devices.device_count', { count: failedCount })}>
+                <span>{`${failedRate} %`}</span>
```
Comment (piotrcichonmee) [inline ota-plus-web/app/reactapp/src/components/campaigns/CampaignSummary.jsx]: Use string templates with "%" from translations.

File: ota-plus-web/app/reactapp/src/helpers/campaignHelper.js [hunk 2]
```diff
@@ -22,46 +22,57 @@ export const getOverallCampaignStatistics = (statistics) => {
   return stats;
 };
 
+const calcRateForStatType = (count, processedCount, percentageFactor) => {
+  if (count === 0) {
+    return 0;
+  }
+  return count / processedCount * 100 < 1
+    ? '< 1'
```
Comment (piotrcichonmee) [inline ota-plus-web/app/reactapp/src/helpers/campaignHelper.js]: Should be moved to translations or at least to const.

File: ota-plus-web/app/reactapp/src/helpers/campaignHelper.js [hunk 3]
```diff
@@ -22,46 +22,57 @@ export const getOverallCampaignStatistics = (statistics) => {
   return stats;
 };
 
+const calcRateForStatType = (count, processedCount, percentageFactor) => {
+  if (count === 0) {
+    return 0;
+  }
+  return count / processedCount * 100 < 1
```
Comment (piotrcichonmee) [inline ota-plus-web/app/reactapp/src/helpers/campaignHelper.js]: Please do not create magic numbers. Add it to const.

File: ota-plus-web/app/reactapp/src/helpers/campaignHelper.js (hunk 2 above)
Comment (piotrsmol) [reply ota-plus-web/app/reactapp/src/helpers/campaignHelper.js]: done

File: ota-plus-web/app/reactapp/src/helpers/campaignHelper.js (hunk 3 above)
Comment (piotrsmol) [reply ota-plus-web/app/reactapp/src/helpers/campaignHelper.js]: done

**Pull request merged**